Finix/Service/Utils/FontExtension.swift

```diff
@@ -0,0 +1,34 @@
+// Created by Valerie N. Prinz on 29/10/2024.
+
+import SwiftUI
+
+extension Font {
+    static var titleLargeSemibold = Font.system(size: 48, weight: .semibold)
```

Comment:
1. Если мы оставляем как есть, то стоит использовать let вместо var. Значения этих переменных не меняются, потому нам не нужно позволять им меняться.
2. В таком варианте эти переменные будут созданы при первом обращении и храниться в памяти до конца работы приложения. В нашем конкретном случае не большая потеря, но всё равно пусть будет тут. Есть ещё вариант использовать computed properties. Таким образом значение будет вычисляться каждый раз по необходимости и удаляться после выхода из скоупа.

Менять ничего не надо, это просто немного деталей о том, как это работает.

```suggestion
    static var titleLargeSemibold { Font.system(size: 48, weight: .semibold) }
```